### DISCUSSION TITLE
Reusable workflows: Failed to fetch

### DISCUSSION THREAD

**Discussion by XSlemX** [community/community#40839] [Actions]
Today,randomly,some workflows triggered failed with message: `...error parsing called workflow "org/repo/.github/worflows/reusable.yaml@main". Failed to fetch workflow.`

This behaviour seems random and the workflow instance that failed cannot be retried. 

This started happening today. We have verified that the reusable workflows do _exists_(they are reused by tens of repos) and they might run if triggered again.

**Comment by weizhoublue** [comment]
there was no any change recent days for the workflow code,  and same issue happened today

**Comment by paoll** [comment]
Seing the exact same behavior here

**Comment by notnmeyer** [comment]
ditto. we've been dealing with these for a couple hours now (the start of our day). i just opened a support ticket about it. same intermittent "Failed to fetch workflow" errors when using the short form too, `uses: ./.github/workflows/blah.yml`.

**Comment by paulvidal** [comment]
Github please fix it, all the merge buttons are green now and CI is not triggering 😱 😱 😱 😱 😱 😱

**Comment by radhi-nasser** [comment]
I am having the same issue as well.

**Comment by tomquist** [comment]
We're seeing the same issue consistently now. It started happening first at Dec 5, 2022, 2:55 PM GMT+1

**Comment by openfinch** [comment]
We're seeing this over a dozen or so different workflows over our private repos. Sometimes no workflows will run, sometimes a handful, sometimes all of them will run. It's been happening for most of the afternoon GMT.

**Comment by steven-karijolich** [comment]
+1 Seeing this issue on my end as well

**Comment by cbeeler** [comment]
My organization is seeing this as well. It is annoying to deal with since another commit must be pushed to the branch to get the workflow to run after this error.

**Comment by wc-whiteheadd** [comment]
we have seen this three times today so far. We have no code changes in that area for 2 months now. Re-running the github action pipeline, often then works but not all the time.

**Comment by notnmeyer** [comment]
things have been working normally for the past couple hours for us 🤞

**Comment by leonardo-minaya** [comment]
anyone having problems yet?

**Comment by jok-dev** [comment]
Having this issue right now.

**Comment by Trinhan87** [comment]
> Today,randomly,some workflows triggered failed with message: `...error parsing called workflow "org/repo/.github/worflows/reusable.yaml@main". Failed to fetch workflow.`
> 
> This behaviour seems random and the workflow instance that failed cannot be retried. 
> 
> This started happening today. We have verified that the reusable workflows do _exists_(they are reused by tens of repos) and they might run if triggered again.

**Comment by Trinhan87** [comment]
A

**Comment by peterbevan-dg** [comment]
Also seeing this happening frequently in our Actions.

We tried re-publishing/re-tagging versions on the assumption/hope that something got corrupted in that process - but still the issue persists so over to you GH :joy:

**Comment by tomquist** [comment]
The issue stopped happening for some time but now we're starting to see it again.

**Comment by arpadgabor** [comment]
We're seeing the same issue lately, and sometimes it works, most of the time it doesn't.

**Comment by rrvs** [comment]
We have the same problem as yesterday, hope it gets fixed soon 😢

**Comment by pavelsurgay** [comment]
We suffer from it too((

**Comment by smokedlinq** [comment]
I have one repo that is experiencing this consistently

**Comment by paulvidal** [comment]
it's not stopping on our side too :( @github-staff when will you consider this one top prio? it's a core feature regression 😢 😭

**Comment by notnmeyer** [comment]
in the support ticket i opened yesterday, someone responded 8 hours ago with, "An issue has been opened for Engineering to investigate."

**Comment by tomquist** [comment]
This is the response I got for my support ticket 16h ago:

> This appears to be a recent, but known issue that Engineering is currently investigating. I've added your voice to that issue and will let you know once I've learned more.

Even though it's a known issue they didn't declare an incident at https://www.githubstatus.com/

**Reply by rrvs** [reply]
same answer in the ticket I opened

**Comment by aramfe** [comment]
This is indeed a VERY CRITICAL problem. This needs to be fixed as fast as possible!

**Comment by jok-dev** [comment]
Just received a reply from github support:

```
Thank you for contacting GitHub Support and sorry for the trouble.

The team is currently investigating the issue.

I'll get back to you as soon as I have an update.
```

**Comment by ThomasMarcelo** [comment]
I've also been getting the `error parsing called workflow` repeatedly for 2 days now. It comes and goes.
I'm referencing a local workflow (private repo) with the `uses: ./.github/workflows/myresusable.yml` reference.

It is random. Retriggering the workflow run sometimes fixes it.

Using the `public` way of referencing the workflow does not work for me because I need to use the workflow from the commit, not a fixed branch or SHA. The `use` parameter doesn't seem to accept variables, making it impossible to do what I need.

**Reply by gbetous-swir** [reply]
Side question : is there a way to retrigg those syntaxically refused workflow ? I usually modify a bit the patch, but did you find a workaround ?

**Reply by ThomasMarcelo** [reply]
No, me and the team are retriggering the workflow runs with this issue by pushing something new to the branch until it works :/

**Reply by gbetous-swir** [reply]
Feature request ! Always being able to retrigg a workflow in case this is a GH bug :smile:

**Comment by gbetous-swir** [comment]
After having modified all the remaining local reference, I now have an issue referencing global workflow with `@master` !

**Comment by tomquist** [comment]
I now officially got the confirmation that this should be resolve through my enterprise support ticket:
> This should now be resolved. We did not post this to status as our SLOs were not breached, so we didn't meet the current threshold for the appropriate incident response process. During this post-mortem, we will be re-evaluating and I will push for a stricter threshold.

**Reply by ThomasMarcelo** [reply]
🤞🏽

**Comment by beyondledtech** [comment]
Absolutely right on point. This was really a nice question to ask. However, Founded in 2008, Beyond LED Technology manufactures and distributes a wide range of [lighting](https://beyondledtechnology.com/) solutions for residential, commercial, and specialty lighting needs. We pride ourselves on being a lighting solutions provider by offering our clients a broad line of innovative and high-quality lighting products.

**Comment by ljrain** [comment]
I am getting this error also, are others still seeing it ? I see the last post was in 2022, this is a new project (demo) that I am working on that is in a public repo and local workflows. Actions are set to allow to run within the repo also.

**Comment by Pictor13** [comment]
Am having this, and to me seems a timing error.  
Like @arpadgabor maybe got into.
As if the depending workflow isn't able to call the other at times, maybe because this latter is already running..  
If this is fast enough to complete, then it can be called again; if not, it fails to read it.  
Like it has some kind of lock on the called-workflow file.

**Reply by Pictor13** [reply]
Otherwise... just check twice you wrote `.yAml` and not `.yml` 😆
